### DISCUSSION TITLE
Welcome to Fluent-for-Steam Discussions!

### DISCUSSION THREAD

**Discussion by purogamer** [purogamer/Fluent-for-Steam#28] [Announcements]

**Comment by bielzin22brhuehue** [comment]
bom theme melhor q ja vi tem um aplicativo que chama Glass2k ele aplica transparencia nas janelas so q na steam ele nao funciona ele aplica so q steam volta pro normal encontra uma solucao ai pra nois 
https://chime.tv/products/glass2k.shtml
